doc/example/group_by.rst

```diff
@@ -0,0 +1,97 @@
+..
+    :copyright: Copyright (c) 2023 ftrack
+
+.. _example/group_by:
+
+*******************************
+Aggregate results with group by
+*******************************
+
+The following aggregation functions are available for summarizing data using
+group by:
+
+:sum():
+
+    Adds together all the values in a particular column.
+
+:avg():
+
+    Calculates the average of a group of selected values.
+
+:min():
+
+    Determines the minimum numeric values in each group.
+
+:max():
+
+    Determines the maximum numeric values in each group.
+
+:count():
+
+    Returns the total number of rows in each group.
+
+To use these methods you must configure API's “strict api” mode when
+generating the session object. Do this by setting :func:`strict_api=True` when
+configuring the session: ::
+
+    session = ftrack_api.Session(
+        server_url='https://YOUR-SITE-URL',
+        api_key='YOUR-API-KEY',
+        api_user='YOUR-API-USER',
+        strict_api=True
+    )
+
+Aggregation functions also require a “group by” clause that specifies how the
+results are aggregated::
+
+    select count(id) from Task group by status_id
+
+Queries that include group by clauses must be executed using the
+:func:`session.call()` method versus :func:`session.query()`.
```

Comment:
Do we want to also include the new version of the help docs [https://help.ftrack-studio.backlight.co/hc/en-us/articles/13129860019863-Operations](url) link as well?